qemu/tests/cfg/pvpanic_basic.cfg

```diff
@@ -0,0 +1,21 @@
+- pvpanic_basic: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = pvpanic
+    disable_shutdown = yes
+    monitor_type = qmp
+    event_check = "GUEST_PANICKED"
+    guest_status = "guest-panicked"
+    driver_name = "pvpanic"
+    timeout = 360
+    backup_image_before_testing = yes
+    restore_image_after_testing = yes
+    set_kdump_cmd = "chkconfig kdump off && systemctl stop kdump.service"
+    RHEL.6:
+        set_kdump_cmd = "chkconfig kdump off && service kdump stop"
+    set_nmi_panic_cmd = "echo 1 > /proc/sys/kernel/unknown_nmi_panic"
+    set_NMICrashDump_cmd = 'wmic class stdregprov call SetDwordValue hDefKey="&h80000002" sSubKeyName="SYSTEM\CurrentControlSet\Control\CrashControl" sValueName="NMICrashDump" uValue=1'
```

Comment:
Can we use the same parameter for both linux and windows?

Reply:
Yes, I will find a generic name.